### PR TITLE
Patch 13

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Updated new EAC UI content. Added new sections " Use the new EAC to ädd an email address" and "Use the new EAC to remove an email address"

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/add-or-remove-email-addresses.md [hunk 1]
```diff
@@ -87,29 +109,20 @@ Set-Mailbox "Dan Jump" -EmailAddresses SMTP:dan.jump@contoso.com,dan.jump@northa
...
-- Run the following command in Exchange Online PowerShell.
+2. In the list of user mailboxes, click the mailbox that you want to remove an email address from. A flyout is displayed for the selected user mailbox.
 
-  ```PowerShell
-  Get-Mailbox <identity> | Format-List EmailAddresses
-  ```
+3. Under the **Mailbox** properties link, click **Manage email address types** under **Email addresses**.
 
-- Verify that the new email address is included in the results.
+4. In the list of email addresses, select the address you want to remove, and then click the delete button.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/add-or-remove-email-addresses.md:121]:
```suggestion
4. In the list of email addresses, select the address you want to remove, and then click the **Delete** button.
```

Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/add-or-remove-email-addresses.md:121]:
```suggestion
4. In the list of email addresses, select the address you want to remove, and then click **Delete**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**